package.json

```diff
@@ -61,6 +61,7 @@
   "homepage": "http://www.d3plus.org",
   "dependencies": {
     "d3": "^3.5.15",
+    "gulp-install": "^0.6.0",
```

Comment:
Sorry, I've commited this line by mistake. Should I remove all the .js builds or just the full.js builds?